### ISSUE TITLE
Release Drafter is *sometimes* including unmerged PRs

### ISSUE THREAD

**Issue by katelovescode** [release-drafter/release-drafter#917]
I'm including a screenshot - release drafter has already added a PR that's open and unmerged; my config is as follows:

```
name-template: 'v$RESOLVED_VERSION 🌈'
tag-template: 'v$RESOLVED_VERSION'
categories:
  - title: '👒 Dependencies'
    labels:
      - 'dependencies'
  - title: '🏕 Backend'
    labels:
      - 'backend'
  - title: '🪔 Frontend'
    label: 'frontend'
change-template: '- $TITLE @$AUTHOR (#$NUMBER)'
change-title-escapes: '\<*_&' # You can add # and @ to disable mentions, and add ` to disable code blocks.
version-resolver:
  major:
    labels:
      - 'major'
  minor:
    labels:
      - 'minor'
  patch:
    labels:
      - 'patch'
  default: patch
template: |
  ## Changes
  $CHANGES
```

From what I can tell there's no reason that that PR should be included; I had to remove another one the other day on 0.17 manually, which I will need to add back in manually, I assume, in this release.  Is there any way to determine why this is happening?

![Screen Shot 2021-08-04 at 9 27 42 AM](https://user-images.githubusercontent.com/8364647/128199275-67bc2e70-2f79-4b62-ab87-348d645899e2.png)

Link to the repo: https://github.com/pieforproviders/pieforproviders

**Comment by jetersen** [comment]
We are simply using the GraphQL api. I cannot see where in the code we should be at fault.

Maybe there is a bug somewhre here: https://github.com/release-drafter/release-drafter/blob/349214e6b3b889d8b333c012cc61a1f1753baf40/lib/commits.js#L63

But 🤷

**Comment by rnorth** [comment]
I think @kiview spotted this with one of his PRs in https://github.com/testcontainers/testcontainers-java, although I'm afraid I can't find any current instance of this...

I'll keep an eye out and try and provide more info if we see it again.

**Comment by jerr0328** [comment]
I thought this was similar to #766 but it might not be. I see the same thing, and I noticed it happens when the action ran last on the pull-request. For instance, the original repo here has this config: https://github.com/pieforproviders/pieforproviders/blob/025bfe99539df5b8660b4c64e972e2331ee1cb78/.github/workflows/release-drafter.yml

I have a similar config on a private repo and noticed that the draft release has an entry for the unmerged PR that last ran. To workaround, I just re-run the last release-drafter workflow on the main branch, and the unmerged PR goes away. I'm using the autolabeler, so I want to keep running on the open PRs.

**Comment by tchupp** [comment]
I am experiencing the same behavior on a newly configured repository.
This run seems to have created the draft release and added the open PR to the list of changes: https://github.com/tchupp/actions-terraform-pr/runs/3456491995?check_suite_focus=true
<img width="700" alt="Screen Shot 2021-08-29 at 5 41 27 PM" src="https://user-images.githubusercontent.com/7855170/131266287-a3af87cc-1bab-4d98-8021-4b135d986999.png">
<img width="608" alt="Screen Shot 2021-08-29 at 5 41 38 PM" src="https://user-images.githubusercontent.com/7855170/131266288-5e9653bc-b722-4f04-b981-d5f045a6b455.png">

**Comment by tchupp** [comment]
I think this PR may be causing this issue: https://github.com/release-drafter/release-drafter/pull/632
The suggested configuration from the [README](https://github.com/release-drafter/release-drafter#usage) includes these triggers:
```yaml
on:
  push:
    # branches to consider in the event; optional, defaults to all
    branches:
      - master
  # pull_request event is required only for autolabeler
  pull_request:
    # Only following types are handled by the action, but one can default to all as well
    types: [opened, reopened, synchronize]
```

The PR trigger is nice because it allows the auto-labeler to work, but I think this is causing the release draft to have unmerged PRs added.

**Comment by tchupp** [comment]
I've been able to work around it using the undocumented input `disable-releaser`:
```
      - uses: release-drafter/release-drafter@v5
        with:
          disable-releaser: github.ref != 'refs/heads/main'
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

**Comment by jetersen** [comment]
Should be solved by #1015 than that would be odd 🗡️
